libs/api/src/lib/graphql/query.public.ts

```diff
@@ -335,7 +335,7 @@ export const GraphQLPublicQuery = new GraphQLObjectType<undefined, Context>({
       type: GraphQLPublicUser,
       description: 'This query returns the user.',
       resolve(root, args, {session}) {
-        return session?.type === SessionType.User ? session.user : null
+        return session?.type === AuthSessionType.User ? session.user : null
```

Comment:
Use `isSessionValid` ?

Reply:
session is undefined if not valid in here

Reply:
Wdym?

I mean this.

```
if (!session) {
  return false
}

return session.type === 'user' ? session.expiresAt > new Date() : true
```

Also the expiration is not considered in the query.public.ts

Reply:
It is.

`session: isSessionValid ? session : null,` in context.ts